packages/resizeImageHttp/index.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const helmet = require('helmet');
+const axios = require('axios');
+const sharp = require('sharp');
+
+const {
+  MAX_THUMB_SIZE = 478,
+  ORIGIN_DOMAIN = 'liker.land',
+  CACHE_TIME_IN_S = 86400,
+  IS_TESTNET,
+} = process.env;
+
+const app = express();
+
+app.use(helmet());
+app.get('/', async (req, res) => {
+  const { url } = req.query;
+  const { origin } = req.headers;
+  if (!IS_TESTNET && origin && !origin.includes(ORIGIN_DOMAIN)) {
```

Comment:
What if the origin header is not in the request?
Should use CORS here to enforce?

Reply:
browser does not enforce cors on img src

Reply:
So the checking is only for best effort?

Reply:
yes unless we check against db record of og image